packages/applets/calculator/src/recordScripts/demo.tsx

```diff
@@ -0,0 +1,167 @@
+import {
+    Controller,
+    declareVideoScript,
+    TitleScreen,
+    Visualizer,
+} from "@launchmenu/applet-lm-recorder";
+import React from "react";
+import {Box, Button, FillBox, UILayer} from "@launchmenu/core";
+import {settings} from "..";
+
+export default declareVideoScript(
```

Comment:
Doesn't look like this actually has anything to do with the calculator in it. Would it be worth including a script for calculator (if anything just `2*5km/hr in m/s` demonstration

Reply:
For sure. We need some simple demo video, I just forgot about this.